pkg/database/database.go

```diff
@@ -132,7 +147,21 @@ func (db *Database) OpenWithCacher(ctx context.Context, cacher cache.Cacher) err
 	var rawDB *gorm.DB
 	if err := withRetries(ctx, func(ctx context.Context) error {
 		var err error
-		rawDB, err = gorm.Open("postgres", c.ConnectionString())
+		driver := ocsql.Wrap(&postgres.Driver{})
```

Comment:
Move this to package init (see my comment above)

Reply:
Done